src/connector/src/source/base.rs

```diff
@@ -592,6 +592,18 @@ impl ConnectorProperties {
             || matches!(self, ConnectorProperties::Azblob(_))
     }
 
+    /// Whether to enable strong schema for the source.
+    pub fn enable_strong_schema(&self) -> bool {
+        let ConnectorProperties::MongodbCdc(mongo) = self else {
+            return false;
+        };
```

Comment:
As a general method on `ConnectorProperties` this feels like _all other connectors do not support strong schema except for mongo_. This is definitely not the truth. The truth is that only mongo defaults to weak schema and requires explicit opt-in.

Consider inline such logic in mongo-specific code, and you can use `try_match_expand!(source_ctx.connector_props, ConnectorProperties::MongodbCdc)?` rather than defaulting to `false`.